### ISSUE TITLE
Non-VR: add option to show both kneeboards at the same time

### ISSUE THREAD

**Issue by OR-WS** [fredemmott/OpenKneeboard#155]
In 2D, the first and second kneeboard (if two kneeboards are enabled in the options) are stacked on top of each other. You can still switch between them and browse to difference tabs/pages on each kneeboard, but you will only ever see one kneeboard.

**Comment by OR-WS** [comment]
A screenshot doesn't allow me to show this. In order to repro:
1. Enable two kneeboards.
2. Turn on the kneeboard(s).
3. Browse to a different page than the first/default page.
4. Use the switch kneeboards command.
5. You'll now see that you're in the second kneeboard which has the first/default page up. Switching again goes back to the first kneeboard of which you changed the page earlier.

**Comment by fredemmott** [comment]
That's still showing me just one kneeboard at a time (as currently expected), not two stacked kneeboards - though which one kneeboard is currently shown does change when you switch them. For example, if you set the non-vr opacity to 50%, you can't see a second kneeboard through it; does this match what you're seeing?

**Comment by fredemmott** [comment]
e.g. what you see in the non-vr game matches what you see in the main openkneeboard window

**Comment by OR-WS** [comment]
> e.g. what you see in the non-vr game matches what you see in the main openkneeboard window

Correct, yes. Stacked might be a confusing word in this, sorry. You see either or indeed. You can cycle between them with the switch command but you'll only see one at the same time.

**Comment by fredemmott** [comment]
Thanks - just wanted to be sure there wasn't something really strange happening that needed fixing too :)

**Comment by OR-WS** [comment]
If I may, if/when this feature makes it into a release, it would be very convenient for 2D users to have a visual indication which of the two kneeboards is the active one that will be manipulated by control inputs. It could be something very simple like a black border, a dot or a hand icon overlaid in one of the corners of the KB, etc.

**Comment by fredemmott** [comment]
The toolbar is only shown on the active kneeboard :)

**Comment by OR-WS** [comment]
> The toolbar is only shown on the active kneeboard :)

Forget I said anything. ;)

**Comment by fredemmott** [comment]
#327 isn't a hard blocker ,but makes sense to do first

**Comment by fredemmott** [comment]
A workaround is to use [the viewer](https://openkneeboard.com/how-to/OpenKneeboard-Viewer/) in combination with the [always on top powertoy](https://learn.microsoft.com/en-us/windows/powertoys/always-on-top), instead of the current non-VR support.

Use:
- the powertoy to make it always on top
- 'f' to switch the background fill to fully transparent
- 'b' for borderless mode
- 'v' to switch to mirroring the VR views instead of only the single non-vr view
- 1-9 to switch which VR view is being mirrored.
